helm/chaos-mesh/Chart.yaml

```diff
@@ -13,14 +13,14 @@
 # limitations under the License.
 #
 apiVersion: v2
-appVersion: "v0.9.0"
+appVersion: "0.0.1"
```

Comment:
IIUC, the `appVersion` will always be replaced by a version specified in the command line? (as commented in https://github.com/chaos-mesh/chaos-mesh/issues/3274#issuecomment-1135547412 ) Then what about using an obviously special version like "0.0.0" or "9999"...

"0.0.1" sounds like an available version (smaller than everyone else)

Reply:
sounds great! `0.0.0` is also a valid sem ver. Updated!